mermin/src/span/opts.rs

```diff
@@ -114,4 +124,7 @@ mod defaults {
     pub fn community_id_seed() -> u16 {
         0
     }
+    pub fn trace_id_timeout() -> Duration {
+        Duration::from_secs(86400) // 24 hours
```

Comment:
Change to from hours and remove the comment.

```suggestion
        Duration::from_hours(24)
```